src/itemChoice/ItemChoice.tsx

```diff
@@ -24,8 +24,8 @@ export type ItemChoiceProps = A11yProps &
     labelInfo?: React.ReactNode
     data?: string
     dataInfo?: string
-    leftAddon?: JSX.Element
-    rightAddon?: JSX.Element
+    leftAddon?: React.ReactNode
```

Comment:
❤️

Reply:
Should you use `ItemProps['leftAddon']`? I don't know actually know what the best practice is here, this will avoid to repeat ourselves for a prop that only passes through to `Item`. WDYT?

Reply:
Let's use a dedicated type maybe (e.g. type LeftAddonType) if we really don't want to duplicate (I found ItemProps['leftAddon'] not really readable).
To be fair, since this ReactNode is internal to this file and the file is still small, i don't feel the urge to create this abstract type for now to reduce this small duplication. Your call folks.

Reply:
IMHO, in the end (it doesn't even matter 🎵 ), we should probably just extend parts of the base `Item` typings. But that's part of another refacto I guess

Reply:
Thanks for the answers 👌  it was more to open the discussion rather than a comment specific to this PR because I see this pattern coming back again a lot.